trace_attributes.go

```diff
@@ -0,0 +1,101 @@
+package opentelemetrygithubactionsjunitreceiver
```

Comment:
centrallise all the attributes for the GitHub workflow events and the Test Suites/Test Cases